meta/main.yml

```diff
@@ -2,15 +2,21 @@
 ---
 galaxy_info:
   author: Dell EMC Networking Engineering 
-  description: The dellos-vrrp role facilitates the configuration of Virtual Router Redundancy Protocol (VRRP) attributes in devices running Dell EMC Networking Operating Systems.
-  license:  Apache 2.0
+  description: >
+       The dellos-vrrp role facilitates the configuration of Virtual Router Redundancy Protocol (VRRP) attributes in 
+       devices running Dell EMC Networking Operating Systems.
+  license: Apache 2.0
   min_ansible_version: 2.2
 
   platforms:
-  - name: dell os9
+  - name: DellOS
 
   galaxy_tags:
     - networking
     - dell 
+    - dellemc
     - emc
-    - os9
+    - dellos9
+    - dellos6
+    - dellos10
+    
```

Comment:
extra new lines were flagged for warnings as well.

Reply:
I have tested it by importing my namespace. its gots 5/5 score
https://galaxy.ansible.com/my-content/namespaces